bool is_good
int32 hamming_dist
int32 id
float64 size
geometry_msgs/PoseStamped pose
